packages/react/src/editor/styles.css

```diff
@@ -104,6 +104,13 @@
   font-family: var(--bn-font-family);
 }
 
+/* Custom block node view wrapper styling */
```

Comment:
why is this changed?

Reply:
This was to do with the blue outline for selected blocks. Previously wasn't working properly for React custom blocks, so this CSS is a fix for that.